lib/internal/event_target.js

```diff
@@ -388,6 +403,31 @@ class Listener {
     }
   }
 
+  get once() {
+    return Boolean(this.flags & kFlagOnce);
+  }
+  get capture() {
+    return Boolean(this.flags & kFlagCapture);
+  }
+  get passive() {
+    return Boolean(this.flags & kFlagPassive);
+  }
+  get isNodeEventTarget() {
```

Comment:
@mabaasit I think the CI failures might come from this typo:

```suggestion
  get isNodeStyleListener() {
```